worker.js

```diff
@@ -155,4 +186,6 @@ app.get('/jwks.json', async (c) => {
 	})
 })
 
+app.all('*', c => {return c.text('Nothing to see here.', 404)})
```

Comment:
Hono by default prints `404 Not Found` on paths that don't exist, so it isn't really necessary to supply our own.